### PR TITLE
fix: replace appcelerator and axway

### PR DESCRIPTION
**Pull request by caspahouzer**
We can leave this open until all copyright things are clear

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,20 +12,20 @@ Learn more about Hyperloop [here](http://www.appcelerator.com/mobile-app-develop
...
 ## Running this app
 
-*BEFORE* attempting to run this application, ensure that you have downloaded and followed the instructions from the appropriate guide below. 
-There are specific prerequisites that must be first met before this application will run properly. Also make sure you [enabled Hyperloop](http://docs.appcelerator.com/platform/latest/#!/guide/Enabling_Hyperloop) before getting started.
+_BEFORE_ attempting to run this application, ensure that you have downloaded and followed the instructions from the appropriate guide below.
+There are specific prerequisites that must be first met before this application will run properly. Also make sure you [enabled Hyperloop](https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Guide/Hyperloop/Enabling_Hyperloop.html) before getting started.
 
-This sample application only runs on the simulator because it uses a demo application GUID. If you want to run this application on device, 
+This sample application only runs on the simulator because it uses a demo application GUID. If you want to run this application on device,
 you need to import the application into your own platform account using `appc new --import`.
```
Comment (m1ga) [inline README.md:19]: remove the `you need to import...` line since that is not needed.

**Review by m1ga — COMMENTED** [review]
just one small note.

File: README.md (hunk 1 above)
Comment (caspahouzer) [reply README.md]: right. it's fixed

**Review by m1ga — APPROVED** [review]

**Comment by caspahouzer** [comment]
I'll merge it for now and will update the copyright afterwards with the coming boilerplate

**Pull request merged**